packages/default-storage-backend/src/helpers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Instead of disabling this rule for the whole file, can we just suppress the lines individually?

Reply:
sure, I don't mind